napari/utils/colormaps/colormap.py

```diff
@@ -236,13 +243,27 @@ def _selection_as_minimum_dtype(self, dtype: np.dtype) -> int:
 
 
 class LabelColormap(LabelColormapBase):
-    """Colormap that shuffles values before mapping to colors.
+    """Colormap color values using cycle of colors .
 
     Attributes
     ----------
-    seed : float
+    colors : ColorArray
+        Colors to be used for mapping.
+        For values above the number of colors,
+        the colors will be cycled.
     use_selection : bool
+        Whether map only selected label.
+        If `True` only selected label will be mapped to not transparent color.
     selection : int
+        The selected label.
+    background_value : int
+        Which value should be treated as a background
+        and mapped to transparent color.
+    interpolation : Literal['zero']
+        required by implementation, please do not set value
+    seed : float
+        seed used for random color generation. Used for reproducibility.
+        It will be removed in the future release.
```

Comment:
This is only used in the labels layer to store the (deprecated) seed, maybe we should remove it in this PR?

Reply:
GH code search returns 0 hits of user code using `colormap.seed`. So I would vote for removing this directly.

Reply:
This seed is a parameter of `label_colormap` function also that is part of the public API for creating colormap. 

The general story behind this change is that until this PR, the colormaps were not part of public API documentation. I try to set up [`autodoc-pydantic`](https://pypi.org/project/autodoc-pydantic/), but it do not work out of the box. So I decided to document missing parameters. 

As we miss many docstrings in our codebase, I prefer to clearly write that argument is deprecated, than left it undocumented.

Reply:
I'm just saying it's not used even after it is stored. So maybe we just remove it from the class, not leave it undocumented. Yes it is a public API but I don't think anyone is using it (and if they are they have plenty of recourse to fix in their own code). But, maybe we can do that in 0.5.0, is that what you would prefer?